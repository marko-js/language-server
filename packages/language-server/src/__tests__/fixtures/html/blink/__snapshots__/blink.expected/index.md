## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <blink>
    |  ^^^^^ Cannot find name 'blink'.
  2 |   This shouldn't be here
  3 | </blink>
```

### Ln 1, Col 2
```marko
> 1 | <blink>
    |  ^^^^^ Fix all of the following:
  Element is visible
  2 |   This shouldn't be here
  3 | </blink>
```

