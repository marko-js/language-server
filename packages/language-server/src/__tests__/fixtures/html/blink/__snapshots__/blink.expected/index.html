<blink data-marko-node-id="0">
  This shouldn't be here
</blink>